src/main/frontend/multi-pipeline-graph-view/multi-pipeline-graph/main/MultiPipelineGraph.tsx

```diff
@@ -39,14 +39,24 @@ export const MultiPipelineGraph = () => {
 
   return (
     <>
-      {Object.entries(groupedRuns).map(([date, runsOnDate]) => (
-        <div className={"pgv-stages__group"} key={date}>
-          <p className="pgv-stages__heading">{date}</p>
-          {runsOnDate.map((run) => (
-            <SingleRun key={run.id} run={run} currentJobPath={currentJobPath} />
-          ))}
+      {Object.keys(groupedRuns).length === 0 ? (
+        <div className="pgv-stages__group">
```

Comment:
also did you want to load it via translations?

Reply:
Would this be a custom translations bundle for our plugin or is there somewhere in core we can take something similar from?

Reply:
custom for this plugin. Generally you don't reuse translations very much from resource bundles so things don't break if they change in core anyway.

Reply:
done